tests/phpunit/StorageExtractorTest.php

```diff
@@ -109,63 +111,41 @@ public function testInvalidTokenPermissions(): void
             'parameters' => [
                 '#token' => getenv('KBC_TEST_WRITE_TOKEN'),
                 'url' => getenv('KBC_TEST_URL'),
+                'tableName' => 'invalid',
             ],
         ];
         $baseDir = $temp->getTmpFolder();
         $fs->dumpFile($baseDir . '/config.json', \GuzzleHttp\json_encode($configFile));
         putenv('KBC_DATADIR=' . $baseDir);
         $app = new Component(new NullLogger());
         self::expectException(UserException::class);
-        self::expectExceptionMessage('The token does not have only read permissions to the bucket in.c-ex-storage');
+        self::expectExceptionMessage('The token must have read-only permissions to the bucket "in.c-ex-storage".');
         $app->run();
     }
 
-    public function testTableFilter(): void
+    public function testTableNoneExistent(): void
     {
         $temp = new Temp('ex-storage');
         $temp->initRunFolder();
         $fs = new Filesystem();
-        $fs->dumpFile($temp->getTmpFolder() . '/tmp.csv', "\"id\",\"foo\"\n\"1\",\"aa\"\n\"2\",\"bb\"\n\"3\",\"cc\"\n");
-        $csv = new CsvFile($temp->getTmpFolder() . '/tmp.csv');
-        $this->client->createTable(getenv('KBC_TEST_BUCKET'), 'some-table-2', $csv, ['primaryKey' => 'id']);
-        $fs->dumpFile($temp->getTmpFolder() . '/tmp.csv', "\"id\",\"bar\"\n\"1\",\"x\"\n\"2\",\"y\"\n\"3\",\"z\"\n");
-        $csv = new CsvFile($temp->getTmpFolder() . '/tmp.csv');
-        $this->client->createTable(getenv('KBC_TEST_BUCKET'), 'some-table-3', $csv, ['primaryKey' => 'id']);
-
         $configFile = [
             'action' => 'run',
             'parameters' => [
                 '#token' => getenv('KBC_TEST_TOKEN'),
                 'url' => getenv('KBC_TEST_URL'),
-                'tables' => ['some-table-3'],
+                'tableName' => 'non-existent-table',
             ],
         ];
         $baseDir = $temp->getTmpFolder();
         $fs->dumpFile($baseDir . '/config.json', \GuzzleHttp\json_encode($configFile));
         putenv('KBC_DATADIR=' . $baseDir);
         $app = new Component(new NullLogger());
+        self::expectException(UserException::class);
+        self::expectExceptionMessage('The table non-existent-table not found in the bucket in.c-ex-storage');
```

Comment:
nešly by tady ty uvozovky okolo?

Reply:
https://github.com/keboola/connection/issues/1354

Reply:
Za mě - pokud existuje možnost, že ten string bude prázdný, tak vy měly být vždycky. Pokud je napevno, tak OK, ale pokud je to věc, která se může měnit, tak uvozovky.

Reply:
@odinuv JO TAKHLE! To je ta exception z KBC, jasně. Sorry :)